docs/api/app.md

```diff
@@ -228,6 +228,10 @@ app.on('login', function(event, webContents, request, authInfo, callback) {
 
 Emitted when the gpu process crashes.
 
+### Event: 'dark-mode-changed' _OS X_
```

Comment:
What do you think about calling this something more generic like `interface-theme-changed` or `platform-theme-changed`?